share/eds/eds.go

```diff
@@ -223,7 +223,7 @@ func prependNamespace(quadrant int, share []byte) []byte {
 	case 0:
 		return append(share[:ipld.NamespaceSize], share...)
 	case 1, 2, 3:
-		return append(appconsts.ParitySharesNamespaceID, share...)
+		return append(namespace.ParitySharesNamespace.Bytes(), share...)
```

Comment:
I looked at the Bytes implementation, and it allocates 😕 
This method will be called on the hottest code path in the node and very likely in the core.
The namespace type should have been a singular field type where we would pick the version and all the other details by directly accessing the subslice through respecting methods.

Reply:
No actionable item here, and just ranting about how this could be better. I will likely touch upon this again during the extraction and either make an issue or fix in place if its an easy fix